src/core/gui/PagePreviewDecoration.cpp

```diff
@@ -27,7 +27,7 @@ void PagePreviewDecoration::drawPageNumberBelowPreview(cairo_t* cr, SidebarPrevi
     xoj::util::CairoSaveGuard saveGuard(cr);
     cairo_text_extents_t extents;
     std::string pageNumber = std::to_string(pageEntry->getIndex() + 1);
-    Color color = control->getSettings()->getBackgroundColor();
+    Color color = control->getSettings()->isDarkTheme() ? Colors::white : Colors::xopp_darkslategray;
```

Comment:
Not sure if one line if-statements are allowed. Couldn't find anything in the coding guideline about that.

Reply:
It's fine as long as it passes the format check.